modules/s2sVendors.js

```diff
@@ -0,0 +1,64 @@
+import { transformBidderParamKeywords } from 'src/utils';
+
+// accountId and bidders params are not included here, should be configured by end-user
+export const S2S_VENDORS = {
+  'appnexus': {
+    adapter: 'prebidServer',
+    cookieSet: false,
+    enabled: true,
+    endpoint: '//prebid.adnxs.com/pbs/v1/openrtb2/auction',
+    syncEndpoint: '//prebid.adnxs.com/pbs/v1/cookie_sync',
+    timeout: 1000
+  },
+  'rubicon': {
+    adapter: 'prebidServer',
+    cookieSet: false,
+    enabled: true,
+    endpoint: '//prebid-server.rubiconproject.com/auction',
+    syncEndpoint: '//prebid-server.rubiconproject.com/cookie_sync',
+    timeout: 500
+  }
+}
+
+export const paramTypes = {
```

Comment:
I don wonder if these belong here. Seems like we've split up transformation in 2 places now (in the bidder `transformBidParams` function and here). I do like these centralized since it's more of a problem with PBS / PBJS being in sync but wondering if they belong here or not.